woocommerce/wc-admin
# Try: Add the settings feature and pages

Fixes #

* Adds the settings feature flags
* Adds the settings pages to WCA and navigation

Coming soon to a repo near you:
* Redirection of old pages
* Actual client-side rendering of pages

### Detailed test instructions:

1. Check that default Settings behavior is unaffected.
2. Toggle the option for settings at `wp-admin/admin.php?page=wc-settings&tab=advanced`
3. With the new navigation off, check that the Settings menu item path is now `wc-admin&path/settings/general` (note that old URLs are temporarily broken)
4. Turn on the navigation feature (you may have to do this manually via `woocommerce_navigation_enabeld` since the settings pages are broken).
5. Check that the settings pages in the new navigation experience are using the new paths.
